### PR TITLE
#28 - added info about Release files to readme

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #28 
Update readme.md with info about Release file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,29 @@ depend on non-free ones. `testing` and `unstable` have the same structure as `ma
...
+Here is the example with minimal fields set:
+
+```text
+Codename: artipie #Repository name
+Architectures: amd64 #Space separated supported architectures set
+Components: main #Repository components set
+Date: Sat, 05 Dec 2020 10:35:57 UTC #Creation date
+SHA256: b91aebb02060c606256206fb6085915f62563a8d78de6ad2873bf171611b5e1c #Checksum of the file
+MD5Sum:
+ 8921ecf7f2fb12bed6bb1125d37c7884 1351 main/amd64/Packages.gz
```
Comment (genryxy) [inline README.md:70]: @olenagerasimova is this field required? In example file [here](https://github.com/artipie/debian-adapter/pull/30/files#diff-64f6819d4f82e0853272b267de2818c2fb7340872cbc8ca3f238cfabf986c935R5) `MD5Sum` is absent.

File: README.md [hunk 2]
```diff
@@ -52,6 +52,29 @@ depend on non-free ones. `testing` and `unstable` have the same structure as `ma
...
+```
+
+The most important fields here are `Architectures` (list of the [architectures](https://wiki.debian.org/SupportedArchitectures) 
+this repository supports separated by space) and `Components` (a whitespace separated list of 
+components or areas, in the layout above components are `main`, `contrib` and `non-free`) as they
```
Comment (genryxy) [inline README.md:75]: @olenagerasimova I do not know exactly is it important but in this paragraph for `Architectures` and `Components` the word `list` is used but in example above the word `set` is used

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see two comments

File: README.md (hunk 1 above)
Comment (olenagerasimova) [reply README.md:70]: @genryxy according to Debian docs - it is [recommended](https://wiki.debian.org/DebianRepository/Format#A.22Release.22_files) and we decided to add `MD5Sum` of the Packages indexes here. But I do not see that this field has an effect on the work of the `apt-get`.

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

File: README.md (hunk 2 above)
Comment (olenagerasimova) [reply README.md:75]: @genryxy hm, these words are not used in pure mathematical context here, I'd say it does not matter

**Comment by olenagerasimova** [comment]
@genryxy could you please take a look on my answers?

**Review by genryxy — APPROVED** [review]
@olenagerasimova yeah, I understood, thanks

**Pull request merged**